cmd/kwil-admin/cmds/setup/peer.go

```diff
@@ -16,7 +16,7 @@ var (
 It will automatically generate required directories and keypairs, and can be given a genesis file and peer list for an existing network.`
```

Comment:
"and peer list for an existing network"  is that what the `--peers` flag was intending to do?  Is this part of the command's docs incorrect?

Reply:
I think so originally, but users should just use the `--chain.p2p.persistent-peers` flag instead. I have updated the example to reflect this